SC_redesign/Crop_and_Soil/field/manure_application.py

```diff
@@ -93,6 +95,10 @@ def apply_machine_manure(self, dry_matter_mass: float, dry_matter_fraction: floa
             Total mass of phosphorus in this application of manure (kg)
         field_coverage : float
             Fraction of the field this manure is applied to (unitless)
```

Comment:
A few missing periods on the params section definitions.
```suggestion
            Fraction of the field this manure is applied to (unitless).
```